reduxfx-view/src/main/java/com/netopyr/reduxfx/vscenegraph/impl/patcher/property/SetterAccessor.java

```diff
@@ -17,7 +17,19 @@
     public void set(Consumer<Object> dispatcher, Object node, String name, VProperty vProperty) {
         if (vProperty.isValueDefined()) {
             try {
-                setter.invoke(node, vProperty.getValue());
+                final Object value = vProperty.getValue();
```

Comment:
I think I found a better solution. It looks as if we can disable varargs, by calling asFixedArity() on the MethodHandle. It seems to work, though I have to admit that I do not fully understand MethodHandles yet. Need to study them thoroughly, there is a lot that can be improved in ReduxFX.

I will update the setter-lookup and after that we can merge the SplitPane change without this workaround.